docs/features/wired-mesh.rst

```diff
@@ -85,3 +85,33 @@ Disable Mesh-on-LAN::
 Please note that this configuration has changed in Gluon 2016.1. Using
 the old commands on 2016.1 and later will break the corresponding options
 in the *Advanced settings*.
+
+Mesh on other interfaces
+========================
+In some cases, you may want to mesh on more interfaces than only LAN or WAN.
+
+This is especially useful if you have multiple other nodes connected to a central node,
+and don't want them to mesh with each other, but only the central node.
+
+In this case, you may need to isolate them via VLANs on the device switch.
+
+Starting from Gluon 2020.2 it's possible to configure additional mesh-interfaces
+by adding an interface configuration with the proto ``gluon_wired``::
+
+  uci set network.mesh_add=interface
+  uci set network.mesh_add.proto=gluon_wired
+  uci set network.mesh_add.index=7
+  uci set network.mesh_add.disabled='0'
+  uci set network.mesh_add.transitive='1'
+  uci set network.mesh_add.ifname='eth2'
+
+``ifname`` is the mesh hardware-interface. This could be a VLAN.
+
+``index`` must be between 0 and 7, and is used to avoid MAC-collisions.
+For details see
+https://github.com/freifunk-gluon/gluon/blob/7fdffcaa1c3cf1b8f5591385d6aa18c15f038f7a/package/gluon-core/luasrc/usr/lib/lua/gluon/util.lua#L139
```

Comment:
You can refer to the page [dev/mac_addresses](https://gluon.readthedocs.io/en/latest/dev/mac_addresses.html) in the docs. Possibly the page should also be extended with additional explanations for a good choice of index.